model.ipynb

kivandeavinash opened the pull request.

kivandeavinash committed:
model.ipynb

ybezginova2016 reviewed (changes requested):
Hey Avinash,

Thank you for your commit.
1) we use Django REST API, please stick to it as well;
2) add load_predict.py in your final commit;
3) to model.ipynb - finalize the model, if you use neural network, please kindly work on two of them - construct one yourself, then you can use a pretrained one.
4) check PEP-8, also  put your imports into the first cell of the notebook;
5) at the end of the notebook, clearly state which model was the best and save it.

kivandeavinash committed:
Add files via upload

kivandeavinash committed:
Delete load_predict.py.txt

kivandeavinash committed:
links

https://www.kaggle.com/datasets/balatmak/newsgroup20bbcnews
https://drive.google.com/file/d/1qBpBPd7UfzBoO0vZkA6EZ8b1dNie0JCz/view?usp=share_link

kivandeavinash committed:
Add files via upload

kivandeavinash committed:
Add files via upload

ybezginova2016 reviewed (approved):
Thank you Avinash, approved and merged.
